src/main.rs

```diff
@@ -781,11 +781,13 @@ mod unit_tests {
                 file: None,
                 mode: ConsoleOutputMode::Null,
                 iommu: false,
```

Comment:
Commit message doesn't follow the correct format - should probably be `vmm:`. Why does Serial Port get capitalised - it's not a proper noun.

Reply:
Fixed. Thanks!